language-reference-manual/lrm.trlc

```diff
@@ -886,7 +886,7 @@ section "Checks" {
               order.'''
   }
 
-  Dynamic_Semantics Check_Evaluation_Order_Across_Blocks {
+  Note Check_Evaluation_Order_Across_Blocks {
```

Comment:
Good point. This is actually also a bug in the linter, and I have created https://github.com/bmw-software-engineering/trlc/issues/36 to fix it.